reference/configuration.md

```diff
@@ -16,7 +16,13 @@ The Ambassador Edge Stack is configured in a declarative fashion, using YAML man
 
 - [`Ingress`](../reference/core/ingress-controller) manifests allows you to use Ambassador as a Kubernetes ingress controller. See the provided documention on configuration with Ambassador, and review the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/) for detailed information on the `Ingress` resource.
 
-Note that each of these `kind`s are supported as both annotations and as Custom Resource Definitions (CRDs).
+- `LogService`manifests help you configure logging. See [Kubernertes Logging Architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/) for more details.
```

Comment:
We should eventually have a page dedicated to `LogService` and link it from here. Searching for it I realized we have none yet.

Reply:
np! I will add it to the list of pages to write!